packages/react-instantsearch-dom-maps/src/GoogleMaps.js

```diff
@@ -145,16 +141,17 @@ class GoogleMaps extends Component {
   }
 
   render() {
-    const { children } = this.props;
+    const { google, children } = this.props;
     const { isMapReady } = this.state;
 
     return (
-      <div className={cx('')}>
-        <div ref={c => (this.element = c)} className={cx('map')} />
-        {isMapReady && children}
-      </div>
+      <GoogleMapsContext.Provider value={{ google, instance: this.instance }}>
+        <GoogleMaps isMapReady={isMapReady} onRef={ref => (this.element = ref)}>
```

Comment:
We can use [`createRef`](https://reactjs.org/docs/react-api.html#reactcreateref) rather than the callback.